newsboat.cpp

```diff
@@ -2,6 +2,7 @@
 #include <errno.h>
 #include <iostream>
 #include <ncurses.h>
+#include <sstream>
```

Comment:
Out of curiosity: what's wrong about spamming `cout`? Given line buffering and how few lines we print out, I don't think this change is about performance. The only real difference I can see is that now messages generated by `print_usage` and `print_version_information` are guaranteed not to be interrupted by anything else, but I don't think that was the intention of your commit.

Reply:
To be honest it's just my personal preference. ;) I've always used streams to build up a string, but I'm ok with reverting this if you don't like it. Not gonna defend it very much.

Reply:
Got it. Let's keep the change, then—I'm fine with it overall.